src/animate/MovieClip.js

```diff
@@ -620,6 +620,32 @@ class MovieClip extends Container {
     }
 
     /**
+     * Short cut for `playSound`
+     * @method PIXI.animate.MovieClip#ps
+     * @param {String} alias The name of the Sound
+     * @param {Boolean} loop The loop property of the sound
+     * @param {MovieClip} context The MovieClip the sound originates from
+     * @return {PIXI.animate.MovieClip}
+     */
+    ps(alias, loop, context) {
```

Comment:
For `playSound` and `ps` you don't need to pass in `context`. You can just do `return this.playSound(alias, loop, this);`